### ISSUE TITLE
vscode cannot connect to azureml compute instance

### ISSUE THREAD

**Issue by LIK2RNG** [microsoft/vscode-tools-for-ai#2033]
Does this occur consistently? No, since this week
Repro steps:
Usually clicking on the VSCode desktop button from my AML compute instance will open the vscode window and set up the connection (in our case via SAML from our company). Now a VSCode window is still opened, but no further authentification steps or any other pop-up messages.

Action: Resolver.resolve
Error type: 72
Error Message: endpoints_resolution_error: Error: could not resolve endpoints. Please check network and try again. Detail: ClientConfigurationError: untrusted_authority: The provided authority is not a trusted authority. Please include this authority in the knownAuthorities config parameter.

Version: 0.28.1
OS: win32
OS Release: 10.0.19045
Product: Visual Studio Code
Product Version: 1.78.0
Language: en

<details>
<summary>Call Stack</summary>

```
s extension.js:2:1981747
async n.acquireBearerToken extension.js:2:1924701
```

</details>

**Comment by gf-eaton** [comment]
I have same error.

**Comment by sevillal** [comment]
Hi @LIK2RNG, I apologize for the issue you are experiencing. Could you please try [this](https://github.com/microsoft/vscode-tools-for-ai/issues/2037#issuecomment-1542774582) and let me know if that helps.

**Comment by sevillal** [comment]
@LIK2RNG VS Code just released version 1.78.2 which should fix the certificates issue, could you please update VS Code (Help > Check for Updates...) and retry the connection.

**Comment by sevillal** [comment]
Closing it for now since other customers have confirmed it is resolved. Please reopen if you still have issues.
